diff --git a/node_modules/react-native-screens/src/screens.d.ts b/node_modules/react-native-screens/src/screens.d.ts
index cbaef2b..9450ecf 100644
--- a/node_modules/react-native-screens/src/screens.d.ts
+++ b/node_modules/react-native-screens/src/screens.d.ts
@@ -26,7 +26,7 @@ declare module 'react-native-screens' {
      * @type "modal" – the new screen will be presented modally. In addition this allow for a nested stack to be rendered inside such screens
      * @type "transparentModal" – the new screen will be presented modally but in addition the second to last screen will remain attached to the stack container such that if the top screen is non opaque the content below can still be seen. If "modal" is used instead the below screen will get unmounted as soon as the transition ends.
      */
-    stackPresentation: StackPresentationTypes;
+    stackPresentation?: StackPresentationTypes;
     /**
      *@description Allows for the customization of how the given screen should appear/dissapear when pushed or popped at the top of the stack. The followin values are currently supported:
      *  @type "default" – uses a platform default animation
